internal/xcrd/crd_test.go

```diff
@@ -111,10 +113,12 @@ func TestForCompositeResource(t *testing.T) {
           "type": "string"
         }
       },
-      "type": "object"
+      "type": "object",
+	  "description": "Status of the resource."
```

Comment:
You might want to format it once

Reply:
Oh, what's going on here is that most of this json object is indented with spaces and I added on tab here, which shows up as a different width in github than in my editor. 

I'll reformat the whole object with tabs instead of spaces, since that seems to be the convention in this repo.

Developers may quarrel about whitespace preferences, but the one thing that everyone should be able to agree on is that a mixture of tabs and spaces on a single line should be forbidden!

Reply:
Agreed